File: src/mispobjects/tag.go
package mispobjects

type Tag struct {
	Id             string `json:"id"`
	Colour         string `json:"colour"`
	Name           string `json:"name"`
	Exportable     bool   `json:"exportable,omitempty"`
	OrgId          string `json:"org_id,omitempty"`
	UserId         string `json:"user_id,omitempty"`
	HideTag        bool   `json:"hide_tag,omitempty"`
	NumericalValue string `json:"numerical_value,omitempty"`
	Count          int    `json:"count,omitempty"`
	AttributeCount int    `json:"attribute_count,omitempty"`
	Favourite      bool   `json:"favourite,omitempty"`
}

type TagForJson struct {
	Tags []Tag `json:"Tag"`
}

File: src/main/kotlin/com/devo/feeds/data/misp/Tag.kt
package com.devo.feeds.data.misp

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Tag(
    val id: String? = null,
    val colour: String? = null,
    val name: String? = null,
    val exportable: Boolean? = null,
    @SerialName("org_id") val orgId: String? = null,
    @SerialName("user_id") val userId: String? = null,
    @SerialName("hide_tag") val hideTag: Boolean? = null,
    @SerialName("numerical_value") val numericalValue: Int? = null,
    val count: Int? = null,
    @SerialName("attribute_count") val attributeCount: Int? = null,
    val favourite: Boolean? = null
)
